src/components/diagrams/networkAreaDiagram/network-area-diagram-content.tsx

```diff
@@ -119,6 +128,7 @@ const dynamicCssRules: CSS_RULE[] = [
         thresholdStatus: THRESHOLD_STATUS.ABOVE,
     },
 ];
+const equipmentsWithPopover = ['LINE', 'TWO_WINDINGS_TRANSFORMER', 'PHASE_SHIFT_TRANSFORMER'];
```

Comment:
use enum value is probably better
```suggestion
const equipmentsWithPopover = [
    EQUIPMENT_TYPES.LINE,
    EQUIPMENT_TYPES.TWO_WINDINGS_TRANSFORMER,
    FEEDER_TYPES.PHASE_SHIFT_TRANSFORMER,
];
```